package.json

```diff
@@ -6,6 +6,7 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
+    "big.js": "^5.2.2",
```

Comment:
This is the incorrect dependency, as you can see from the [Pathwright for this milestone](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896094/), the correct package is [big-js](https://www.npmjs.com/package/big-js), not big.js, please do correct this, an you are going to be able to import big without an stickler error.